qa/suites/fs/upgrade/mds_upgrade_sequence/tasks/0-from/reef.yaml

```diff
@@ -0,0 +1,31 @@
+meta:
+- desc: |
+   setup ceph/reef
+
+tasks:
+- install:
+    branch: reef
+    exclude_packages:
+      - ceph-volume
+- print: "**** done install task..."
+- cephadm:
+    image: quay.ceph.io/ceph-ci/ceph:reef
+    roleless: true
+    compiled_cephadm_branch: reef
```

Comment:
the good news is this is actually working in terms of pulling in the reef binary. The bad news is the upgrade still fails with
```
Upgrade: Paused due to UPGRADE_BAD_TARGET_VERSION: Upgrade: cannot upgrade/downgrade to 18.0.0-5596-gdb1309a8
```
I think because it considers an upgrade from 18.2.0 -> 18.0.0 an unsupported downgrade. Pretty sure we didn't ad quincy as the new basedpoint for upgrades in the reef cycle until main was already reporting v18 so we didn't have this issue. Will need to see if there's some workaround we can use for this case.

Reply:
Decided to move the reef start point work into https://github.com/ceph/ceph/pull/53105 so that we can get this through to at least get the upgrades to start from quincy instead of pacific